cmd/wallet-sdk-gomobile/docs/usage.md

```diff
@@ -837,6 +841,10 @@ let credentials = interaction.requestCredential(withPreAuth: didDocument.asserti
 
 let issuerURI = interaction.issuerURI() // Optional (but useful)
 
+interaction.requireAcknowledgment() // return true if the issuer requires acknowledgment that credentials are accepted or rejected by the user.
+try interaction.acknowledgeSuccess() // acknowledgment issuer that the user accepted credentials.
+try interaction.acknowledgeReject() // acknowledgment issuer that the user rejected credentials.
```

Comment:
We need to pass the reject reason to the issuer (need a string arg here). We can push it as a follow up PR.